docs/actions.html

```diff
@@ -519,7 +519,7 @@ <h2 id="part-4"><strong>Part 4: Input and Output constraints</strong></h2>
 
     <br /><span style="margin-left: 20px;"><strong><em>request</em></strong></span>
 
-    <pre>POST https://example.com/products/ipod/buy</span><br/>
+    <pre>POST https://example.com/products/ipod/buy</span>
```

Comment:
Should this closing span not be removed?

Reply:
Eagle eye! Resolved.